templates/jvm.options.j2

```diff
@@ -41,15 +41,37 @@
 -XX:CMSInitiatingOccupancyFraction=75
 -XX:+UseCMSInitiatingOccupancyOnly
 
+## G1GC Configuration
+# NOTE: G1GC is only supported on JDK version 10 or later.
+# To use G1GC uncomment the lines below.
+# 10-:-XX:-UseConcMarkSweepGC
+# 10-:-XX:-UseCMSInitiatingOccupancyOnly
+# 10-:-XX:+UseG1GC
+{% if es_version is version('7.3.0', '<=') %}
```

Comment:
Is this configuration related to the Elasticsearch version or Java version? I suspect it is related to the Java version which could cause issues for a user that has `es_java_install` set to something other than the default.

Making assumptions about the version is probably ok as long as there is also a way for the user to opt out or override the setting.

Reply:
I just realized that these lines are all commented out. So maybe the templating should just be removed altogether?

Reply:
>Is this configuration related to the Elasticsearch version or Java version? 
This configuration is related to Elasticsearch version, it's based on the default [jvm.options] file provided by Elasticsearch RPM/DEB package. I deployed all minor versions from 6.7 to 7.5 on centos/debian containers and compared config files like `jvm.options` to checked all differences when there are some.

Note, that lines prefixed by a number `10-:`, `8:`, ... are relative to only a specific Java version (java 10, java 8, ...) (more details [here](https://www.elastic.co/guide/en/elasticsearch/reference/7.4/jvm-options.html)).

>I just realized that these lines are all commented out. So maybe the templating should just be removed altogether?

This commented lines are from the default files from RPM/DEB packages, I added them to be the closest has possible to the package files. However we can remove commented lines if you think it's not necessary.